hypertension/dc-7101-algorithms/py-root/lib/algorithms/continuous_medication.py

```diff
@@ -1,4 +1,16 @@
-from .utils import hypertension_medications
+hypertension_medications = {
```

Comment:
This python set was moved from the utilities file (since it's only used in this file)